docs/using_packet.md

```diff
@@ -181,27 +182,38 @@ Invoking Syntex From Your Build Script
 --------------------------------------
 
 To pull everything together, we need to invoke `syntex` within the build script.
-This is a simple build script for building just a single packet type; see the
-`build.rs` from `libpnet` for an example of how to build a bunch of them:
+This is a simple build script for recursively building files using the pnet_macros; see also the
+`build.rs` from `libpnet` as an example of how many packets are built:
```

Comment:
I will re-write this section first, but revert to the original if this example usage is seen as too dense.